scripts/shelley-testnet.sh

```diff
@@ -13,8 +13,6 @@ set -e
 # CMD="stack exec --nix --"
 CMD="cabal new-run --"
 
-. $(dirname $0)/lib-node.sh
-
```

Comment:
@CodiePP, I think that change should break the script -- this lib defines `nodeargs`, which is used below.

Reply:
it's included twice. one time would be enough?

Reply:
Oh, right, my bad! : -)